packages/react-native/ReactAndroid/src/test/java/com/facebook/react/packagerconnection/JSPackagerClientTest.kt

```diff
@@ -0,0 +1,137 @@
+package com.facebook.react.packagerconnection
+
+import com.facebook.react.packagerconnection.ReconnectingWebSocket.ConnectionCallback
+import java.io.IOException
+import okio.ByteString
+import okio.ByteString.Companion.encodeUtf8
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
```

Comment:
```suggestion
import org.mockito.Mockito.`when` as whenever 
```

so you don't have to use backticks for `when` in the code.Y
You can check the linked PR in the umbrella issue to have an example on how to do it.